packages/cli-kit/src/private/node/ui/components/Prompt.test.tsx

```diff
@@ -34,7 +34,8 @@ describe('Prompt', async () => {
 
     expect(renderInstance.lastFrame()).toMatchInlineSnapshot(`
       "?  Associate your project with the org Castile Ventures?
-      [36m✔[39m  [36msecond[39m"
```

Comment:
When I see these tests I always wonder if we should do matches without ASCI escape characters. As a code reviewer, `[36m` says nothing to me. I'd have to go and search what the escape code means. Nothing actionable in for this PR though, just a random thought that I had.

Reply:
That's a good idea. In a future iteration we could provide a custom matcher that outputs in the terminal the two outputs so that you can see the styles applied.

Reply:
Actually if you get an error right now it already works, you will see the colored output in the terminal since the escape sequences are interpreted.